dao/src/main/kotlin/tables/runs/evaluator/RuleViolationsTable.kt

```diff
@@ -34,11 +34,11 @@ import org.jetbrains.exposed.sql.and
  * A table to represent a rule violation.
  */
 object RuleViolationsTable : SortableTable("rule_violations") {
-    val rule = text("rule")
-    val packageIdentifierId = reference("package_identifier_id", IdentifiersTable).nullable()
+    val rule = text("rule").sortable()
+    val packageIdentifierId = reference("package_identifier_id", IdentifiersTable).sortable().nullable()
```

Comment:
Not sure if it makes sense to make this sortable, IIUC it would just sort by the numeric id, not the string representation of the identifier.